### PR TITLE
SAR-6381 - Store BV result to DB

### PR DESCRIPTION
**Pull request by Daveo-G**

### CODE REVIEW

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/BusinessVerification.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incorporatedentityidentificationfrontend.models
+
+import play.api.libs.json.{Json, OFormat}
+
+case class BusinessVerification(verificationStatus: String)
```
Comment (kabuumu) [inline app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/BusinessVerification.scala:21]: Could this be an ADT with a trait of BusinessVerificationState then overridden with the possible states?

It's a constant set of states so we could model it as something a little more compiler friendly.

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/services/IncorporatedEntityInformationService.scala [hunk 2]
```diff
@@ -64,4 +70,5 @@ object IncorporatedEntityInformationService {
   val ctutrKey: String = "ctutr"
   val companyNumberKey: String = "company_number"
```
Comment (kabuumu) [inline app/uk/gov/hmrc/incorporatedentityidentificationfrontend/services/IncorporatedEntityInformationService.scala:71]: This isn't used.

**Review by kabuumu — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/BusinessVerification.scala (hunk 1 above)
Comment (Daveo-G) [reply app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/BusinessVerification.scala]: Added an ADT

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/services/IncorporatedEntityInformationService.scala (hunk 2 above)
Comment (Daveo-G) [reply app/uk/gov/hmrc/incorporatedentityidentificationfrontend/services/IncorporatedEntityInformationService.scala]: Removed unused key

**Pull request merged**